messages/en.json

```diff
@@ -2,106 +2,202 @@
         "Resigned to the inevitable, \u0002{0}\u0002 is led to the gallows. Once the twitching stops, it is discovered that the village lynched a{1} \u0002{2}\u0002.",
+        "Resigned to the inevitable, \u0002{0}\u0002 is led to the gallows. Afterwords, the village inspects the body and finds a{1} \u0002{2}\u0002.",
+        "Resigned to the inevitable, \u0002{0}\u0002 is led to the gallows. Unfortunately the rope snaps, and the \u0002{2}\u0002 hits the ground and dies.",
         "Before the rope is pulled, \u0002{0}\u0002, a{1} \u0002{2}\u0002, pulls the pin on a grenade. They hesitate, and it explodes, killing them.",
         "Before the rope is pulled, \u0002{0}\u0002, a{1} \u0002{2}\u0002, throws a grenade at the mob. The grenade explodes early.",
+        "Before the rope is pulled, \u0002{0}\u0002 shouts to the crowd: \"Stop! I'm really a{1} \u0002{2}\u0002!\", but it is too late.",
+        "Before the rope is pulled, \u0002{0}\u0002, a{1} \u0002{2}\u0002, shouts angrily at the crowd.",
+        "The villagers drag \u0002{0}\u0002 to the edge of a cliff. Before they can protest, the unfortunate \u0002{2}\u0002 slips and falls into the abyss.",
         "Uttering a chilling war cry, the villagers drag \u0002{0}\u0002 to the edge of a cliff and throw them off. The cliff face is now covered with the leftover remains of the late \u0002{2}\u0002.",
+        "The villagers angrily behead \u0002{0}\u0002, discovering they have decapitated a{1} \u0002{2}\u0002.",
+        "\u0002{0}\u0002 is thrown to a pack of hungry animals. The \u0002{2}\u0002 doesn't last long.",
+        "The village decides to feed \u0002{0}\u0002 to a den of lions. Soon nothing but bones are left of the late \u0002{2}\u0002.",
         "For SCIENCE, the villagers throw \u0002{0}\u0002 into a volcano. They discover the melting point of a{1} \u0002{2}\u0002 is less than that of lava.",
-        "The angry villagers grab \u0002{0}\u0002 by the legs and arms, and throw them into a large fire. It seems that a{1} \u0002{2}\u0002's body doesn't burn that well, after all."
+        "For SCIENCE, the villagers electrocute \u0002{0}\u0002. They discover the fried body of a{1} \u0002{2}\u0002.",
+        "For SCIENCE, the villagers give \u0002{0}\u0002 an acid bath. They discover that a{1} \u0002{2}\u0002 is too weak to avoid decomposition.",
+        "The villagers tie \u0002{0}\u0002 to a bunch of balloons. They watch as the \u0002{2}\u0002 floats off into the sunset.",
+        "The village decides to exile \u0002{0}\u0002. After they pack up and leave, nobody ever sees the \u0002{2}\u0002 again.",
+        "The angry villagers grab \u0002{0}\u0002 by the legs and arms, and throw them into a large fire. It seems that a{1} \u0002{2}\u0002's body doesn't burn that well, after all.",
+        "The angry villagers tie \u0002{0}\u0002 to a pole and burn them at the stake. It seems that a{1} \u0002{2}\u0002's body doesn't burn that well, after all."
     ],
```

Comment:
Generic review for lynch messages: If their role is to be revealed, we can't "lose" the body, or else it's impossible to identify. One thing to pay attention to.

Reply:
the existing volcano one goes against this, and is also just as graphic as some of the ones you objected to IMO (and then there's the BBQ endgame message, so cannibalism by villagers isn't automatically too far either). i don't think your points are *wrong* but a lot of existing messages would need to be reworked

Reply:
Fair enough, I didn't think we had this issue in the bot already. We could fix it but I really don't care enough, let's just do whatever :P